bootstrap/bootstrap.sh

```diff
@@ -28,8 +29,54 @@ sudo apt-get update
 sudo apt-get -y dist-upgrade
 
 #----nodejs install
-curl -sL https://deb.nodesource.com/setup_7.x | sudo -E bash -
+curl -sL https://deb.nodesource.com/setup_6.x | sudo -E bash -
 sudo apt-get install -y nodejs
 
-#----alsa install
-sudo apt-get install -y alsa-base alsa-utils
+#----install official requirements
+sudo apt-get install -y alsa-base alsa-utils libasound2-dev
+
+#----install missing pigpio in Raspbian Lite (the command npm install pigpio will be exec by package.json)
+sudo apt-get install pigpio
+
+#----install git and download tjbot
+if [ ! -d "${TJBOT_FOLDER}/recipes/conversation" ]; then
+    sudo apt-get install -y git
+
+    rm -Rf /home/pi/git/tjbot
+    git clone https://github.com/ibmtjbot/tjbot.git /home/pi/git/tjbot
+
+    TJBOT_FOLDER='/home/pi/git/tjbot'
+fi
+
+#----install conversation (to install it will resolve tjbotlib and other dependencies)
+cd $TJBOT_FOLDER
+cd recipes/conversation
+echo "path: $PWD"
+
+sudo npm install --unsafe-perm -g node-gyp
```

Comment:
@fivdi I also just format my SD card and installed Raspbian Jessie Lite 2017-04-10 to test in a clean rpi the command `npm install` without sudo and other params.
Result: 
[bootstrap.txt](https://github.com/ibmtjbot/tjbot/files/949700/bootstrap.txt)

It is similar with your results but I have an extra issue:
```
gyp WARN EACCES user "root" does not have permission to access the dev dir "/root/.node-gyp/6.10.2"
gyp WARN EACCES attempting to reinstall using temporary dev dir "/home/pi/tjbot/recipes/conversation/node_modules/fibers/.node-gyp"
```

In general it is solved with `--unsafe-perm` but I didn't used it to keep in the same level of you.

As we can see in our logs, if we use only `npm install` we receive a lot of warnings in installation process. I don't know if it is related with a wrong package version or other thing.
Do you know how to solve these warning messages? I know it is only warning but is a lot of then and give the impression of an installation problem.

As we see I only could avoid that using -g parameter. You have a suspect that it install only a specific package as global and not install package.json. For me it install the -g parameter as global **and also** package.json as local because the folder `node_modules` is always created inside the conversation and tests folder.

BTW, do you know any other way to avoid these warning messages?